docs/Options.md

```diff
@@ -44,9 +44,10 @@ Default: `null`
 ### capabilities
 Defines the capabilities you want to run in your WebDriver session. Check out the [WebDriver Protocol](https://w3c.github.io/webdriver/#capabilities) for more details. If you run an older driver that doesn't support WebDriver you need to apply the [JSONWireProtocol capabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities) to successfuly run a session. Also a useful utility is the Sauce Labs [Automated Test Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/)
 that helps you to create this object by clicking together your desired capabilities.
+Note: `Object[]` is only used when using the wdio testrunner.
 
-Type: `Object`<br>
-Default: `{ browserName: 'firefox' }`<br>
+Type: `Object`|`Object[]`<br>
```

Comment:
As I mentioned, we should move this to WDIO options and keep `Object` as type. Under WDIO options we can put capabilities and mention to have it set as Object (for multiremote) or Object[]

Reply:
to clarify, 
__WebDriver Options__ has a capabilities option that has type `Object`
__WDIO Options__ has also a capabilities option from type `Object|Object[]` , first when run as multiremote